docs/ossm/ossm2-vs-ossm3.md

```diff
@@ -110,6 +110,19 @@ The TLS protocol version can be set through [Istio Workload Minimum TLS Version
 
 `auto mTLS` is enabled by default in both OpenShift Service Mesh 2 and OpenShift Service Mesh 3.
 
+
+## Certificate Authority Configuration Changes
+
+OpenShift Service Mesh 3 significantly simplifies certificate authority (CA) configuration compared to OpenShift Service Mesh 2. While basic functionality for configuring external CAs like cert-manager is preserved, many of the fine-grained controls previously available in the `ServiceMeshControlPlane` resource have been removed.
+
+Users requiring custom CA configurations should use Istio's built-in CA configuration options or integrate with external certificate management systems.
+
+## Proxy Configuration Changes
+
+OpenShift Service Mesh 3 removes several proxy configuration options that were available in OpenShift Service Mesh 2's `ServiceMeshControlPlane` resource to align more closely with upstream Istio. Protocol auto-detection configuration through `proxy.networking.protocol.autoDetect` has been removed, with the service mesh now using Istio's default protocol detection behavior.
```

Comment:
We could mention that the only setting that still exists in upstream is MeshConfig's `protocolDetectionTimeout` - all the others have been removed upstream